### PR TITLE
4 View a list of all reviews

### PR DESCRIPTION
**Pull request by nasben64**

### CODE REVIEW

File: src/components/Reviews.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      setReviews(reviewsFromApi);
+    });
+    setIsLoading(false);
+  }, []);
+
+  console.log(isLoading);
+
+//   if (isLoading) {
+//     return <h2>Looading ....</h2>;
+//   }
```
Comment (yalicee) [inline src/components/Reviews.jsx]: remove this

File: src/components/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Home = () => {
+  return (
+    <div>
+      <h2 className="header-text">NC Games Review </h2>
+    </div>
+  );
+};
+
```
Comment (yalicee) [inline src/components/Home.jsx]: need a link to reviews on home if youre going to have it on a /reviews path

File: src/context/ReviewsContext.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React, { useState, createContext } from "react";
+
+export const ReviewsContext = createContext();
+
+export const ReviewsContextProvider = ({ children }) => {
+  const [reviews, setReviews] = useState([]);
+  return (
+    <ReviewsContext.Provider value={{ reviews, setReviews }}>
+      {children}
```
Comment (yalicee) [inline src/context/ReviewsContext.js]: What is this doing?

**Review by yalicee — CHANGES_REQUESTED** [review]
Good work so far :) See my comments and message me on slack please

File: src/context/ReviewsContext.js (hunk 3 above)
Comment (nasben64) [reply src/context/ReviewsContext.js]: this stores the reviews in the context so I don't need to pass it down every time I need to use it

File: src/components/Home.jsx (hunk 2 above)
Comment (nasben64) [reply src/components/Home.jsx]: I am thinking later on I may display the top 10 reviews on the home page

File: src/components/Reviews.jsx (hunk 1 above)
Comment (nasben64) [reply src/components/Reviews.jsx]: ok done

File: src/context/ReviewsContext.js (hunk 3 above)
Comment (yalicee) [reply src/context/ReviewsContext.js]: okay

Comment (yalicee) [reply src/context/ReviewsContext.js]: Do you need all of your reviews in the header or every other component?

**Review by yalicee — APPROVED** [review]
Okay please continue ahead :)

**Comment by nasben64** [comment]
I have the SingleReview which uses this reviews array to render itself on the Reviews component if it's a bad design I can remove it and just use the useState inside the Reviews component, just this is what I had on my wireframe

**Pull request merged**